pkg/ingester/kafka_consumer.go

```diff
@@ -42,11 +43,17 @@ func newConsumerMetrics(reg prometheus.Registerer) *consumerMetrics {
 			Help:                        "The latency of a push request after consuming from Kafka",
 			NativeHistogramBucketFactor: 1.1,
 		}),
+		consumeWorkersCount: promauto.With(reg).NewGauge(prometheus.GaugeOpts{
+			Name: "loki_ingester_partition_consume_workers_count",
+			Help: "The number of workers that are consuming from Kafka",
```

Comment:
Nit this is not consuming, but processing **already** consumed records.

```suggestion
			Help: "The number of workers that are processing records from Kafka",
```